app/views/squads/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= label_tag t(:new, attr: Squad.model_name.human), nil, class: 'reg-header' %>
```

Comment:
This is nitpicking, but ```label``` is used to refer to form controls. You probably want <hx>